go.mod

```diff
@@ -39,6 +39,11 @@ require (
 	gorm.io/plugin/dbresolver v1.4.7
 )
 
+require (
+	github.com/go-sql-driver/mysql v1.7.0 // indirect
+	gorm.io/driver/mysql v1.4.7 // indirect
+)
+
```

Comment:
was that introduced by `datatypes` package?

Reply:
yes